_posts/2020-08-26-bazel-3.5.md

```diff
@@ -0,0 +1,108 @@
+-   b37c51c7085f0aefe04034dd451acb847605ddb5: Add include_prefix and strip_include_prefix to cc_common.compile
+-   f6ad35fcde93f92c591778ed7db38d167f5bbc03: Delete --experimental_transparent_compression
+-   39bc97eab295bddb35b38bfc4a2ff3d2b15d034e: Remove --experimental_action_args
+-   b9706675a7abf6ceebb250f0b3dfa4087a0c35f6: Stop needlessly parsing WORKSPACE files from external repositories.
+-   e574d558da17cfd0f818e7a937a07926aa270069: Allow hyphen char in workspace name
+-   9993785fa0c4fa4172aa31d306f3abea76833abf: Allow dot ('.') in workspace names.
+
+## Incompatible changes
+
+- Removed the `--experimental_process_wrapper_wait_fix` flag.
+- Removed the `--experimental_ui_deduplicate` flag.
+- Removed the `--experimental_transparent_compression` flag.
+- Removed the `--experimental_action_args` flag.
+- Bazel now correctly prefers Xcode versions in `/Applications` over
+  any other paths, which resolves an issue with accidentally picking up
+  an Xcode version from a Time Machine backup or network disk. In the
+  improbable case that you relied on the old behavior and Bazel now picks
+  up Xcode from the wrong location, you can fix it by moving that Xcode
+  version to /Applications.
+
+## New features:
+
+- `cquery` now follows aspects when the `--include_aspects` flag is used.
+- Add support to bazel/crosstool for building arm64 on macos.
+
+## Important changes:
+
+- Fixed resource shrinking when \<overlayable/> tags are used.
+- Remove legacy handling of `--extra_checks`
+- Remove old incompatible flag `--incompatible_symlinked_sandbox_expands_tree_artifacts_in_runfiles_tree`.
+- Update coverage configuration for Python, filegroup, and shell script
+  rules to distinguish between source and dependency attributes.
+
+- Add `InstrumentedFilesInfo` provider to Starlark globals.
+- Make filegroup always forward `InstrumentedFilesProvider` and not
+  collect any sources directly.
+- Major changes to reporting of Starlark errors and the call stack.
+  (Please be alert to possible regressions, such as errors that lack
+   relevant location information.)
+- Dot ('.') is now allowed in workspace names. See [#11837](https://github.com/bazelbuild/bazel/issues/11837).
+
+## Rules
+- `cc_common.compile` support for `include_prefix` / `strip_include_prefix`.
+- Support signing key rotation in android_binary.
+- `--apple_bitcode` now takes an optional platform and only applies the
+  Bitcode mode to that platform if present. The option may be provided
+  multiple times.
+- Add opt in `oso_prefix_is_pwd` feature for Apple builds.
+- NS_BLOCK_ASSERTIONS is now passed for all Apple architectures.
+- Java 14 is now supported through custom `toolchain_jdk_14`. The `java_tools`
+  repository provides a new release with [JDK 14 java tools](
+  https://github.com/bazelbuild/java_tools/releases/tag/javac14_v1.0).
+  To build with toolchain 14 on Mac Os X, add this to your `WORKSPACE` file:
```

Comment:
This information (the full code sample) should be moved to the documentation or somewhere discoverable.
The release notes should have a link.

Reply:
I filed a bug against the toolchain to document it at the link we already have.